versions.yml

```
@@ -321,25 +320,25 @@ modGroups:
       homepage: https://modrinth.com/mod/betterf3
       modrinthId: 8shC1gFX
       desc: Makes the F3-menu customizable and cleaner
-      url: https://cdn.modrinth.com/data/8shC1gFX/versions/sPcACJ3r/BetterF3-8.0.2-Fabric-1.20.2.jar
+      url: https://cdn.modrinth.com/data/8shC1gFX/versions/cWQJKCXe/BetterF3-8.0.3-Fabric-1.20.2.jar
       configUrl: https://github.com/MineInAbyss/launchy-mods/releases/download/v1.0.0/BetterF3_1.0.zip
 
     - name: Desired Servers
       license: MIT
       homepage: https://modrinth.com/mod/desired-servers
       modrinthId: GDpe4LHD
       desc: Ship servers without modifying servers.dat
-      url: https://cdn.modrinth.com/data/GDpe4LHD/versions/6WCSTVyB/desiredservers-fabric-1.20.2-1.3.0.jar
+      url: https://cdn.modrinth.com/data/GDpe4LHD/versions/ul5Tx6jd/desiredservers-fabric-1.20.2-1.3.1.jar
       configUrl: https://github.com/Wynntils/launchy-config/releases/download/v0.0.1-rc.0/DesiredServers.zip
       forceConfigDownload: true
 
   Misc Enhancements:
-    - name: First Person Models
+    - name: First Person Model
```

```suggestion
    - name: First Person Models
```

Same. If the name is wrong, it stays for now. We can eventually put them into "removed" with the old name and add it back with the correct name.